src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/LocaleStage.php

```diff
@@ -71,7 +71,7 @@ public function isNecessary()
             if (($locale !== false) && (in_array($locale, $installedLanguages))) {
                 $this->writeParams(array('locale' => $locale));
 
-                return false;
+                return true;
```

Comment:
Well this is installer and default locale for website should not be chosen without "option" to change it specially if it is detected from browser. For example let say you have de in browser but site still should be in en... you can't go back Language chooser is not displayed. This setting should be true for any case where there is more than one language installed.